tmt-app/src/Components/InfoLinkPage.jsx

```diff
@@ -102,13 +138,30 @@ function MajorPage() {
                       }}
                     >
                       {item.info.map((item, index) => {
-                        return (
+                        return typeof item === "string" ? (
                           <ListItemText
-                            key={`${item.name}-${index}`}
+                            key={`${item.label}-${index}`}
```

Comment:
if typeof item === 'string', then accessing item.label will throw an error

Reply:
Great catch! I didn't even notice because it was working with any error.